integration/testdata/https/client-cert.pem

```diff
@@ -1,73 +1,75 @@
 Certificate:
     Data:
         Version: 3 (0x2)
-        Serial Number: 3 (0x3)
-    Signature Algorithm: sha1WithRSAEncryption
-        Issuer: C=US, ST=CA, L=SanFrancisco, O=Fort-Funston, OU=changeme, CN=changeme/name=changeme/emailAddress=mail@host.domain
+        Serial Number:
+            83:eb:91:a7:38:2d:dc:1c:2e:0e:4f:44:b6:20:83:86
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: O = Docker, CN = Docker
         Validity
-            Not Before: Dec  4 14:17:54 2013 GMT
-            Not After : Dec  2 14:17:54 2023 GMT
-        Subject: C=US, ST=CA, L=SanFrancisco, O=Fort-Funston, OU=changeme, CN=client/name=changeme/emailAddress=mail@host.domain
+            Not Before: May 17 11:24:18 2021 GMT
+            Not After : May 17 11:24:18 2031 GMT
+        Subject: O = Docker
```

Comment:
I guess this one should have a `CN=client` (at least) to make it work. Need to find out why easycert doesn't include that for the client certificate (probably because it depends on the hostname), so perhaps back to using `openssl` to generate